tablo/models.py

```diff
@@ -966,19 +966,13 @@ def copy_data_table_for_import(dataset_id):
         sequence_name=sequence_name
     )
 
-    index_command = 'CREATE INDEX {table_name}_geom_index ON {table_name} USING gist({column_name})'.format(
```

Comment:
Also nice to see this go away. Is indexing somehow accounted for in the `populate_point_data` function, or does the index just serve no useful purpose?

Reply:
Actually, it's still there. See https://github.com/consbio/tablo/blob/pandas/tablo/models.py#L1052
The index command that's removed from here wasn't doing anything.